.github/workflows/verify-fleetd-base.yml

```diff
@@ -63,6 +63,31 @@ jobs:
           diff fleetd-base.msi fleetd-base-permalink.msi
           curl -o fleetd-base-permalink.pkg "$(jq --raw-output '.fleetd_base_pkg_url' meta.json)"
           diff fleetd-base.pkg fleetd-base-permalink.pkg
+      - name: Slack Notification
+        if: failure()
+        uses: slackapi/slack-github-action@e28cf165c92ffef168d23c5c9000cffc8a25e117 # v1.24.0
+        with:
+          payload: |
+            {
+              "text": "${{ job.status }}\n${{ github.event.pull_request.html_url || github.event.head.html_url }}",
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "Verify fleetd-base files > ${{ github.job}} result: ${{ job.status }}\nhttps://github.com/fleetdm/fleet/actions/runs/${{
```

Comment:
Let's distinguish the text so that it's easier to know which one of the three failed (by looking at the Slack message only), something like:
- "fleetd-base checksums verification failed", for the first one.
- "Installation test of fleetd-base.msi failed", for the second one.
- "Installation test of fleetd-base.pkg failed", for the third one.

Reply:
Absolutely. The "${{ github.job }}` portion of the text should take care of that, unless I've missed something:
<img width="652" alt="Screenshot 2024-12-26 at 2 01 13 PM" src="https://github.com/user-attachments/assets/d208c9f8-999e-4c0a-a818-5e72570481ab" />

Reply:
Ah cool. I missed that.